Tests/test_utils.c

```diff
@@ -30,10 +33,72 @@ TEST_GROUP_RUNNER(Utils) {
     RUN_TEST_CASE(Utils, Utils_LittleEndianSerializeDeserialize);
 }
 
+TEST(Utils, Utils_QuickUint32Pow10) {
+    uint32_t result = 0U;
+    bool success = false;

```

Comment:
Not needed. We can use TEST_ASSERT_TRUE/FALSE instead of TEST_ASSERT. In my opinion, it looks better.